repo/blob/storage.go

```diff
@@ -56,9 +56,9 @@ type OutputBuffer interface {
 // Capacity describes the storage capacity and usage of a Volume.
 type Capacity struct {
 	// Size of volume in bytes.
-	SizeB uint64
+	SizeB uint64 `json:"capacity,omitempty"`
 	// Available (writeable) space in bytes.
-	FreeB uint64
+	FreeB uint64 `json:"available,omitempty"`
```

Comment:
nit; Wondering if it makes sense to set `omitempty` on `FreeB` ? What happens when the volume is full ?

Reply:
Yes, a 0 value for `FreeB` should be included.
```suggestion
	FreeB uint64 `json:"available"`
```

Reply:
That's a good point: we wouldn't be able to distinguish that from when FreeB is unset (unbounded). I think it should be enough for just `*Capacity` to be omitted.